components/raftstore-v2/src/operation/ready/apply_trace.rs

```diff
@@ -543,6 +559,24 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
         let apply_trace = self.storage_mut().apply_trace_mut();
         apply_trace.on_flush(cf, index);
         apply_trace.maybe_advance_admin_flushed(apply_index);
+        let stale_ssts = self.sst_apply_state().stale_ssts(cf, index);
+        if stale_ssts.is_empty() {
+            return;
+        }
+        info!(
+            self.logger,
+            "delete stale ssts after flush";
```

Comment:
I think it's somewhat misleading. Maybe "schedule delete stale ssts after flush".

Reply:
yesy, fixed it.